src/MovieCard/MovieCard.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
-import Rating from '../Rating/Rating.js';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 import './MovieCard.css';
 
-
-const MovieCard = ({ movie, history, userId }) => {
+const MovieCard = ({ movie, userId }) => {
   return (
-    <section className='movieCard' onClick={() => {
-      history.push(`/${movie.id}`)}}>
+    <section className='movieCard'>
       <h1 className='cardTitle'>{movie.title}</h1>
       <h4>Average Rating: {parseFloat(movie.average_rating).toFixed(1)}</h4>
       <h4>{movie.userRating && `Your Rating: ${movie.userRating.rating}`}</h4>
       <h4>{!movie.userRating && userId && `Click to add rating`}</h4>
       <h4>{!movie.userRating && !userId && `Sign in to add your rating`}</h4>
-      <img className='poster' src={movie.poster_path} alt={movie.title + ' movie poster'}/>
+      <Link to={`/${movie.id}`}>
```

Comment:
I like the link on the image instead of on the entire card. It's a bit more intuitive. Can we possibly implement this into the logo at the top of page as well to get it to route back to the homepage? just a thought!

Reply:
Oh that is an excellent idea! We should definitely implement clicking the logo to take the user back to the home page.